crates/wac-graph/src/encoding.rs

```diff
@@ -215,7 +215,7 @@ impl<'a> TypeEncoder<'a> {
     }
 
     fn func_type(&self, state: &mut State, id: FuncTypeId) -> u32 {
-        log::debug!("encoding function {id}");
+        log::debug!("encoding function type");
```

Comment:
Might make sense to just delete the log or to demote it to a trace log. The same goes for similar logs where essentially the same log is emitted at the beginning of the encoding function and at the end.

Reply:
Given the recursive nature of the type system, I do prefer having an entry/exit log in these places to see what we've started encoding before encoding any recursive types it contains.